api/tests/integration/infrastructure/repositories/assessment-repository_test.js

```diff
@@ -680,4 +680,169 @@ describe('Integration | Infrastructure | Repositories | assessment-repository',
       });
     });
   });
+
+  describe('#getLastPlacementAssessmentByUserIdAndCourseId', () => {
+
+    const userId = 42;
+    const courseId = 'rec23kfr5hfD54f';
+
+    afterEach(async () => {
+      await databaseBuilder.clean();
+    });
+
+    it('should return null if nothing found', () => {
+      // when
+      const promise = assessmentRepository.getLastPlacementAssessmentByUserIdAndCourseId(userId, courseId);
+
+      // then
+      return expect(promise).to.be.fulfilled
```

Comment:
qu'apporte le `to.be.fulfilled` ? j'ai l'impression il est redondant avec l'assertion sur le résultat

Reply:
C'était pour avoir une assertion en amont sur la résolution de la promesse, plutôt que d'avoir juste une erreur si ça pète ... .

Reply:
Enlevé, et remplacé par des await